CodeStarterCamp_Week5/Hacker.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Hacker.swift
+//  CodeStarterCamp_Week5
+//
+//  Created by Hyejeong Jeong on 2022/11/16.
+//
+
+import Foundation
+
+struct Hacker {
+    func hackPassedApplicantsList() {
+        // applicantsList.passedApplicantsList.append(mySon)
```

Comment:
해커가 합격자 명단을 변경할 수 없네요 👍

Reply:
```suggestion
    func hackPassedApplicantsList() {
        print(applicantsList.passedApplicantsList)
```

이 작업은 어떻게 될까요?

Reply:
```suggestion
    func hackPassedApplicantsList() {
        print(applicantsList.totalApplicantsList)
```

이 작업은 어떻게 될까요?

먼저 생각해보신 후 직접 실행해서 확인해보세요 🙂

Reply:
1. `print(applicantsList.passedApplicantsList)`
-> `private`라 컴파일 오류가 발생할 것이라 생각했습니다. 실행했을 때도 컴파일 오류가 발생했습니다!
2. `print(applicantsList.totalApplicantsList)`
-> 읽기 전용으로 배열 값이 출력될 것이라 생각했는데 출력되었습니다~!